js/index.js

```diff
@@ -0,0 +1,54 @@
+$(function() {
+	
+
+// 1. Listen for click 
+function 
+	$.on("submit")() {
```

Comment:
@mjhrich You forgot to select the element that you are performing the function on. ```$(' the element').submit(function() {
 .... code ....
})``` 🔨